app/src/main/java/cm/aptoide/pt/repository/request/RequestFactory.java

```diff
@@ -106,6 +107,12 @@ public GetStoreWidgetsRequest newStoreWidgets(String url) {
     return this.getStoreWidgetsRequestFactory.newStoreWidgets(url, googlePlayServicesAvailable);
   }
 
```

Comment:
Same as AdsRepository, would create a class in your flavour where i would add your new methods.